JS.md

```diff
@@ -0,0 +1,235 @@
+# JS API
+
+In the MVP, the only way to access WebAssembly on the Web is through an explicit
+JS API which is defined below. (In the future, WebAssembly may also be
+loaded and run directly from an HTML `<script type='module'>` tag&mdash;and any other
+Web API that loads ES6 modules via URL&mdash;as part of 
+[ES6 Module integration](Modules.md#integration-with-es6-modules).)
+
+## The `Wasm` object
+
+The `Wasm` object is the initial value of the `Wasm` property of the global
+object. Like the `Math` and `JSON` objects, the `Wasm` object is a plain
+JS object (not a constructor or function) that acts like a namespace and has the
+following properties:
+
+### Constructor Properties of the `Wasm` object
+
+* `Wasm.Module` : see [`Wasm.Module` objects](#wasmmodule-objects)
+* `Wasm.Instance` : see [`Wasm.Instance` objects](#wasminstance-objects)
+* `Wasm.CompileError` : a [NativeError](http://tc39.github.io/ecma262/#sec-nativeerror-object-structure)
+   which indicates an error during WebAssembly decoding or validation
+* `Wasm.RuntimeError` : a [NativeError](http://tc39.github.io/ecma262/#sec-nativeerror-object-structure)
+   which indicates an error while running WebAssembly code
```

Comment:
ECMAScript doesn't differentiate within broad classes of errors, but it is a permitted extension of implementations. Wasm can differentiate whatever it wants.

My question would be, what benefit do we hope to get from standardizing the distinction? Do we expect users to switch on the different errors? Or, do we expect errors to get better over time with more detailed implementation work (which standardizing too early might inhibit)?